lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -846,14 +846,34 @@ def index_invocations(self, trans: GalaxyWebTransaction, workflow_id=None, **kwd
         else:
             # Get all invocation if user is admin
             user_id = target_user_id
-
+        sort_by = kwd.get('sortBy') or kwd.get('sort_by')
```

Comment:
Can we get away with just sort_by (and limit) or is there some Vue component that can't be configured to do that?